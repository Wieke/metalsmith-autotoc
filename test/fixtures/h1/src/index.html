---
title: 'hoge'
autotoc: true
template: 'layout.eco'
---
<h1>aaa</h1>
<p>paragraph</p>

<h2>bbb</h2>
<p>paragraph</p>

<h2>ccc</h2>
<p>paragraph</p>

<h1>ddd</h1>
<p>paragraph</p>
