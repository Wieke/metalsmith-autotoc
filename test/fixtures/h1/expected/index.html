



<ol class="toc">
  
  <li><a href="#aaa">aaa</a>
    
<ol class="toc">
  
  <li><a href="#bbb">bbb</a>
    
  </li>
  
  <li><a href="#ccc">ccc</a>
    
  </li>
  
</ol>

  </li>
  
  <li><a href="#ddd">ddd</a>
    
  </li>
  
</ol>



<h1 id="aaa">aaa</h1>
<p>paragraph</p>

<h2 id="bbb">bbb</h2>
<p>paragraph</p>

<h2 id="ccc">ccc</h2>
<p>paragraph</p>

<h1 id="ddd">ddd</h1>
<p>paragraph</p>

